build-gluster-org/jobs/glusto-tests-lint.yml

```diff
@@ -8,10 +8,10 @@
     scm:
     - git:
         branches:
-        - $GERRIT_BRANCH
-        refspec: $GERRIT_REFSPEC
+        - $GITHUB_BRANCH
+        refspec: $GITHUB_REFSPEC
         choosing-strategy: gerrit
```

Comment:
Can you please change `choosing-strategy`

Reply:
Ah! Nice catch misses it. Thanks for pointing it out.

Reply:
Done